### PR TITLE
Oper_testing_merge into master

### PR DESCRIPTION
**Pull request by LaenExplain**

### CODE REVIEW

File: opercap.f90 [hunk 1]
```diff
@@ -0,0 +1,422 @@
...
+    function GFFI_A_oper(w,vesc,A,mq)
+        double precision :: p, mu,w,vesc,u,muplus,mN,A,Ei,B
+        double precision :: dgamic,GFFI_A_oper
+        integer :: mq
+        p = mdm*w
+        mu = mdm/mnuc/A
+        muplus = (1.+mu)/2.
+        u = sqrt(w**2-vesc**2)
+        mN = A*mnuc
+        Ei  = 5.8407d-2/(mN*(0.91*mN**(1./3.)+0.3)**2)
```
Comment (aaronvincent) [inline opercap.f90]: To be consistent with Catena, should be: 
`Ei = 1./(2.*mN*2.671223d2/(45.d0*a**(-1./3.) - 25.d0*a**(-2./3.)))`

File: opercap.f90 [hunk 2]
```diff
@@ -0,0 +1,422 @@
...
+                            a_shared = a !make accessible via the module
+
+                            mu = mdm/(mnuc*a)
+                            muplus = (1.+mu)/2.
+                            muminus = (mu-1.d0)/2.
+
+                            J = AtomicSpin_oper(eli)
+                            !!!!!!!!!!!!!!!!!!!!!!!!!simplify yConverse mathematical opperation to help loop speed?
+                            ! yConverse = 2/3.*((0.91*(mnuc*a)**(1./3)+0.3)*10**-13)**2/(2*hbar*c0)**2 !the conversion factor between q and y: y = yConverse * q^2
+                            yConverse = (1.38E-27) * ( ((mnuc*a)**(1./3) + 0.33) / (hbar*c0) )**2 !should be less computationally demanding I think, unless the 10**-27 makes floats do weird stuff - could combine the 10**-27 with the (hbar*c0)**2 to get a less extreme float prefactor
```
Comment (aaronvincent) [inline opercap.f90]: To be consistent with Catena should be
`yConverse = 2.671223d2/(45.d0*a**(-1./3.) - 25.d0*a**(-2./3.))`

**Review by aaronvincent — CHANGES_REQUESTED** [review]

File: opercap.f90 (hunk 1 above)
Comment (LaenExplain) [reply opercap.f90]: We found that it should be:
`Ei = 1./4.d0/mN/264.114*(45.d0*A**(-1./3.)-25.d0*A**(-2./3.))`

File: opercap.f90 (hunk 2 above)
Comment (LaenExplain) [reply opercap.f90]: We found it should be:
`yConverse = 264.114/(45.d0*A**(-1./3.)-25.d0*A**(-2./3.))`

**Pull request merged**